### PR TITLE
CE-3198: Add insight usage (actionable or informative) to display dif…

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Add insight usage (actionable or informative) to display different data (insights count)

### CODE REVIEW

File: extensions/wikia/InsightsV2/models/InsightsFlagsModel.php [hunk 1]
```diff
@@ -6,8 +6,8 @@
  */
 
 class InsightsFlagsModel extends InsightsModel {
-
 	const INSIGHT_TYPE = 'flags';
+	const INSIGHT_USAGE = InsightsModel::INSIGHTS_USAGE_INFORMATIVE;
```
Comment (kamilkoterba) [inline extensions/wikia/InsightsV2/models/InsightsFlagsModel.php]: What about moving INSIGHT_USAGE value to config?

File: extensions/wikia/InsightsV2/InsightsHelper.php [hunk 2]
```diff
@@ -198,7 +198,9 @@ public function prepareInsightsList( $limit = 0 ) {
 			$insightsList[$key] = [
 				'subtitle' => self::INSIGHT_SUBTITLE_MSG_PREFIX . $key,
 				'description' => self::INSIGHT_DESCRIPTION_MSG_PREFIX . $key,
-				'count' => $this->prepareCountDisplay( $insightsCountService->getCount( $key ) ),
+				'count' => $class::INSIGHT_USAGE == InsightsModel::INSIGHTS_USAGE_ACTIONABLE
```
Comment (kamilkoterba) [inline extensions/wikia/InsightsV2/InsightsHelper.php]: And as for moving INSIGHT_USAGE to config, we could retrieve it's state from config here.

Comment (lukaszkonieczny) [inline extensions/wikia/InsightsV2/InsightsHelper.php]: Actually I don't move it to config to avoid creating every insight model here. What do you think?

Comment (kamilkoterba) [inline extensions/wikia/InsightsV2/InsightsHelper.php]: Discussed offline. In general I'm for code cleanness. Performance gain is most likely neglectable here.

**Comment by kamilkoterba** [comment]
LGTM

**Comment by azarek** [comment]
LGTM

**Pull request merged**